pyscript.core/tests/integration/test_01_basic.py

```diff
@@ -93,6 +93,19 @@ def test_print(self):
         )
         assert self.console.log.lines[-1] == "hello pyscript"
 
+    @skip_worker("input does not raise exceptions in workers")
```

Comment:
this should be a `@only_main` IMHO.

In my view, "skip" means "this test currently fails but only temporarily, we will fix it"

Reply:
Done